api/src/main/java/org/openmrs/module/openconceptlab/importer/Importer.java

```diff
@@ -171,13 +179,42 @@ public void run() {
 
 					@Override
 					public void run() throws IOException {
-						in = new CountingInputStream(inputStream);
-
-						importService.updateSubscriptionUrl(anImport, IMPORTED_VIA_API);
+						if (in != null) {
+							if (!in.equals(inputStream)) {
+								setInputStream(inputStream);
+							}

```

Comment:
@rkorytkowski I don't remember what was purpose of this, I've removed these conditions and it works fine anyway.